src/server/server_family.cc

```diff
@@ -426,6 +492,17 @@ void SlowLog(CmdArgList args, ConnectionContext* cntx) {
   (*cntx)->SendError(UnknownSubCmd(sub_cmd, "SLOWLOG"), kSyntaxErrType);
 }
 
+void FlushEntireDb(Transaction* transaction) {
```

Comment:
You could use `Drakarys` or `FlushAll` which are already present in this file :)